jiracli/password.go

```diff
@@ -43,6 +50,21 @@ func (o *GlobalOptions) GetPass() string {
 			if err != nil {
 				panic(err)
 			}
+		} else if o.PasswordSource.Value == "gopass" {
+			if o.PasswordDirectory.Value != "" {
+				orig := os.Getenv("PASSWORD_STORE_DIR")
+				os.Setenv("PASSWORD_STORE_DIR", o.PasswordDirectory.Value)
+				defer os.Setenv("PASSWORD_STORE_DIR", orig)
+			}
+			if bin, err := exec.LookPath("gopass"); err == nil {
```

Comment:
Maybe a warning here for the failure case of gopass not being found? My understanding of the flow is that we'd drop back to the default password behaviour